src/main/java/edu/ucsb/cs156/happiercows/services/SystemInfoServiceImpl.java

```diff
@@ -22,21 +27,41 @@ public class SystemInfoServiceImpl extends SystemInfoService {
   @Value("${app.showSwaggerUILink:false}")
   private boolean showSwaggerUILink;
 
+  @Value("${app.startQtrYYYYQ:20243}")
+  private String startQtrYYYYQ;
+
+  @Value("${app.endQtrYYYYQ:20244}")
+  private String endQtrYYYYQ;
+
   @Value("${app.sourceRepo}")
-  private String sourceRepo = "https://github.com/ucsb-cs156/proj-happycows";
+  private String sourceRepo = "https://github.com/ucsb-cs156-s24/proj-happycows-s24-4pm-5";
```

Comment:
```suggestion
  private String sourceRepo = "https://github.com/ucsb-cs156/proj-happycows";
```

This needs to stay as the default.  Please override the default by setting SOURCE_REPO in your .env and/or your dokku config:set variables.

Your code has to be, eventually, mergeable into https://github.com/ucsb-cs156/proj-happycows. and this change isn't.